src/hooks/useSherClaimContract.ts

```diff
@@ -2,10 +2,12 @@ import { useContract, useProvider } from "wagmi"
 import SherClaimInterface from "../abi/SherClaim.json"
 import { SherClaim } from "../contracts/SherClaim"
 
+export const SHER_CLAIM_ADDRESS = process.env.REACT_APP_SHER_CLAIM_ADDRESS as string
```

Comment:
Would be nice if you could add the newly used env variables to the `.env.example`, but other than that, this is good to go 🚀

Reply:
yeah, good call